### DISCUSSION TITLE
Extra's are on when spawning car

### DISCUSSION THREAD

**Discussion by Boomkipje** [matsn0w/IIS-EmergencyLS-ELS-FiveM#143] [Q&A] [answered]
When I spawn in a car (not all of them) there are some lights on static. I think this is in the vehicles.meta, but I am not sure how to fix it. Any idea?

**Comment by matsn0w** [comment, marked as answer]
Hey, thank you for reaching out!

This is most likely caused by your vehicle spawner. It will probably spawn your vehicle with some random extras enabled. That's nice for some vehicles to come up with some variation, but undesirable for ELS. So unfortunately I can't do much about it. Which spawner are you using?

Also see #136

**Reply by Boomkipje** [reply]
Hi, I am simply typing *car xxx* in the console and i assume it's the same with any job garage.

**Reply by matsn0w** [reply]
Yeah, probably. I'm affraid you can't really do anything about it. One option to consider is using/creating a custom spawner which doesn't randomize the extra's or allows you to pre-configure it.
